test/namedtuple.jl

```diff
@@ -201,6 +201,15 @@ end
 abstr_nt_22194_3()
 @test Base.return_types(abstr_nt_22194_3, ()) == Any[Any]
 
+@test Base.delete((a=1, b=2), (:b,)) == (a=1,)
```

Comment:
I think I read that the result here should be inferrable? If so, should these test also check that?

Reply:
How do I test that?

Reply:
What gets inferred?

Reply:
`@inferred` should work.

Reply:
Also `Base.` isn't needed, right?